docs/FAQs.md

```diff
@@ -29,9 +29,9 @@ When an organization contains a small amount of data, the backfilling process is
 
 ![Edit GitHub settings](./images/edit-github-settings.png)
 
-<h3>Q: What about pull requests and issues? I noticed I need to grant read and write permissions. Why is this needed?</h3>
+<h3>Q: What about pull requests, contents and issues? I noticed I need to grant read and write permissions. Why is this needed?</h3>
 
-**A:** This is needed so our app can create links to Jira issues from pull request or issue comments. When you create a comment and include the issue key surrounded by square brackets, our app while ping Jira to see if that issue key exists in a project in Jira and, if it finds a matching issue, will create a link for easy navigation.
+**A:** This is needed so our app can create links to Jira issues from pull request or issue comments. When you create a comment and include the issue key surrounded by square brackets, our app while ping Jira to see if that issue key exists in a project in Jira and, if it finds a matching issue, will create a link for easy navigation. As for contents, we need the write access so we can create a branch on your behalf.
```

Comment:
I'd've changed that to something like "by your request", otherwise this may sound if the app would be creating branches whenever they wish...

Reply:
acceptable